batch.go

```diff
@@ -274,6 +276,10 @@ func (batch *Batch) Validate() error {
 	return errors.New("use an implementation of batch or NewBatch")
 }
 
+func (batch *Batch) SetValidation(opts *ValidateOpts) {
```

Comment:
I don't know if we want to add these options onto `File` and `Batcher`, but it would let us bypass validation to handle arbitrary Origin/Identification values.